src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -7373,28 +7373,7 @@ static void Main(string[] args)
         }
     }
 }";
-            var expected = @"class Test
-{
-    void method()
-    {
-        static void Main(string[] args)
-        {
-            int v = 0;
-            for(int i=0 ; i<5; i++)
-            {
-                v = NewMethod(v, i);
```

Comment:
well, IDE assumes code is most of time in broken state.

so, let's say I am coding

I have code

void Method()
{
...
}

and I started to add method above it. note missing "}" for my new method.

void newMethod () {

void Method() { ..}

and then realized I want to share a code in existing Method(), and I want to extract that code out as new method and call it inside of my new method void NewMethod() { ...

this is common scenario. fix is for a bug opened.

this is more realistic scenario than extract method only working when everything is in perfect state in IDE.

if we want to disable this scenario, we at least make sure when this fail, we let users know, they have to close "}" previous method for extract method to work. 

otherwise, extract method becomes user unfriendly feature.